### ISSUE TITLE
relation to zetasql project

### ISSUE THREAD

**Issue by tmc** [cloudspannerecosystem/memefish#64]
How does this project relate to https://github.com/google/zetasql and https://github.com/goccy/go-zetasql ?

**Comment by makenowjust** [comment]
TLDR: This project is unrelated to [google/zetasql] and [goccy/go-zetasql].

[google/zetasql]: https://github.com/google/zetasql
[goccy/go-zetasql]: https://github.com/goccy/go-zetasql

[google/zetasql] is the SQL analyzing framework developed by Google. ZetaSQL is similar to Spanner SQL, but there are some differences between them. [goccy/go-zetasql] is the Go binding of ZetaSQL.

On the other hand, memefish is the parser library for Spanner SQL written in Go.

When developing memefish, I sometimes refer to the ZetaSQL implementation because the Spanner implementation is not open access. But, the relation is nothing more.
